salt/planet/config/nginx.planet.conf.jinja

```diff
@@ -19,8 +19,10 @@ server {
     server_name {{ site }};
     error_log /var/log/nginx/{{ site }}.error.log;
     access_log /var/log/nginx/{{ site }}.access.log;
-    ssl_certificate /etc/lego/certificates/{{ grains['fqdn'] }}.crt;
```

Comment:
I think we can completely remove both of these server blocks (80, 443)

Reply:
Looking at current DNS we are still hitting the bugs host directly, so I don't think so.